source/reference/operator/query/rand.txt

```diff
@@ -0,0 +1,60 @@
+=====
+$rand
+=====
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. query:: $rand
+
+   *Syntax*: ``{ $rand: {} }``
```

Comment:
Please use the same syntax code block as you do in source/reference/operator/aggregation/rand.txt

i.e.:

:query:`$rand` has the following syntax:

   .. code-block:: javascript

      { $rand: {} }

Reply:
Done. 

The syntax styles are very different both within and across the query and aggregation groups. Perhaps we need to agree on a standard form.